packages/api/src/router/event-group.ts

```diff
@@ -133,25 +140,19 @@ export const eventGroupRouter = router({
           promises.push(
             service
               .createEvent({
-                id: ev.id,
                 title: ev.title,
                 attendeeEmails: [...attendeeEmails],
                 startDate: ev.startDate,
                 endDate: ev.endDate,
               })
               .then(async ({ event }) => {
-                // Update microsoft id of event
-                if (event.microsoftId) {
-                  // We save microsoftId to our DB to be able to delete the event later
-                  // We don't need this in Google since we can overwrite created event ID in Google
-                  await ctx.prisma.event.update({
-                    where: { id: ev.id },
-                    data: {
-                      microsoftId: event.microsoftId,
-                    },
-                  });
-                }
-                return event;
+                // Update event with the provider's event id
+                return await ctx.prisma.event.update({
+                  where: { id: ev.id },
+                  data: {
+                    providerEventId: event.providerEventId,
+                  },
+                });
```

Comment:
Since Google doesn't accept our id's, I've let them set the id themselves, which we then update in our db.

We already did this for Microsoft but now we do it for both.

Maybe calendarId is a better name?

Reply:
Yes, when I see `id` on the model, I wouldn't expect that it's some third-party settings these but postgres (via our prisma default value).

Reply:
Or are you talking about `providerEventId` -- unsure 🤔 

if the event is actually for the entire calendar, then it shouldn't be called `*EventId`. How does google generate the id? Do you have a link to the relevant piece in the docs maybe? I'll take a look just to get context.

Reply:
It's the providerEventId which is created by Google/Microsoft. The id is generated by prisma 

https://developers.google.com/calendar/api/v3/reference/events/insert#request-body